src/book/02-system/02-subsystems/04-motion.mdx

```diff
@@ -6,17 +6,81 @@ description: Motion and how it works in the NUbots codebase.
 slug: /system/subsystems/motion
 ---
 
-The motion subsystem contains modules that are responsible for walking, kicking, moving the head, and running scripts. They perform these motions by sending servo commands (called targets) to the motors.
+This page details motion modules within the NUbots main codebase, such as the walk engines and the Script Engine. The Script Engine is detailed on this page, while Script Runner and Script Tuner can be found on the [Behaviour](/system/subsystems/behaviour#script-runner) page.
+
+
+## Walk Engine
+
+Currently NUbots has two walk engines. One is an **open loop hybrid walk controller** and the other is an **open loop walk** based on **quintic splines**. These engines take care of the mathematics and control involved in taking steps without falling over. The walk engines do not plan where to walk to, only how to walk.
+
+A walk engine will trigger when a [**walkcommand**](https://github.com/NUbots/NUbots/blob/master/shared/message/motion/WalkCommand.proto) is [emitted](/system/foundations/nuclear#emit-statements). The walk command gives a vector [x, y, z] which represents:
```

Comment:
kinda weird bolding walkcommand and not emitted. I would bold both or neither. if you accept my "splines" link suggestion, probably follow the same convention for it too, in terms of boldness